### PR TITLE
Add support for S3 PublicAccessBlock

### PR DESCRIPTION
**Pull request by mtp1376**
PublicAccessBlock is needed for deployment of a static website:
https://docs.aws.amazon.com/AmazonS3/latest/userguide/website-hosting-custom-domain-walkthrough.html#root-domain-walkthrough-configure-bucket-permissions

### CODE REVIEW

File: site/blog/2022/deploy-static-website.mdx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
```
Comment (aguillenv) [inline site/blog/2022/deploy-static-website.mdx:1]: is this file part of this PR?

File: src/modules/aws_s3/entity/public_access_block.ts [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+@Entity()
+export class PublicAccessBlock {
+  /**
+   * @public
+   * Block public access to buckets and objects granted through new access control lists (ACLs)
+   */
+  @Column({
+    default: true,
+  })
+  BlockPublicAcls?: boolean;
```
Comment (aguillenv) [inline src/modules/aws_s3/entity/public_access_block.ts:27]: Can we keep entity props camelCased for consistency?

File: src/modules/aws_s3/entity/public_access_block.ts [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+ *
+ * @example
+ * ```sql TheButton[Make the S3 Bucket Public]="Make the S3 Bucket Public"
+ * UPDATE public_access_block SET BlockPublicAcls = false, IgnorePublicAcls = false, BlockPublicPolicy = false, RestrictPublicBuckets = false WHERE bucket_name = 'mybucketname';
+ * ```
+ *
+ * @see https://docs.aws.amazon.com/AmazonS3/latest/userguide/access-control-block-public-access.html
+ */
+@Entity()
+export class PublicAccessBlock {
```
Comment (aguillenv) [inline src/modules/aws_s3/entity/public_access_block.ts:19]: can we add the auto-incremented id?

File: site/blog/2022/deploy-static-website.mdx (hunk 1 above)
Comment (mtp1376) [reply site/blog/2022/deploy-static-website.mdx:1]: I'll remove it

File: src/modules/aws_s3/entity/public_access_block.ts (hunk 2 above)
Comment (mtp1376) [reply src/modules/aws_s3/entity/public_access_block.ts:27]: You're right. I'll change to camelCase.

File: src/modules/aws_s3/entity/public_access_block.ts (hunk 3 above)
Comment (mtp1376) [reply src/modules/aws_s3/entity/public_access_block.ts:19]: Here there's a 1-1 relationship between `Bucket` and `PublicAccessBlock` so I believe `bucket` field is enough (and has a unique constraint - it's primary key).

File: site/blog/2022/deploy-static-website.mdx [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+slug: deploy-static-website
+title: Deploy a Static Website Using SQL
+date: 2022-12-21
+unlisted: true
```
Comment (depombo) [inline site/blog/2022/deploy-static-website.mdx:5]: I don't think this works. You can ignore it in the docusaurus config if you would like though. We do it for the clean aws account one

File: src/modules/iasql_functions/iasql.ts [hunk 5]
```diff
@@ -1019,7 +1019,7 @@ async function commitApply(
           if (r.diff.entitiesInDbOnly.length > 0) {
             logger
               .scope({ dbId })
-              .info(`${name} has records to create`, { records: r.diff.entitiesInDbOnly });
+              .info(`${name} has records to create - apply`, { records: r.diff.entitiesInDbOnly });
```
Comment (dfellis) [inline src/modules/iasql_functions/iasql.ts:1022]: What's with these changes?

**Review by aguillenv — APPROVED** [review]

**Review by depombo — APPROVED** [review]

**Review by dfellis — APPROVED** [review]

**Pull request merged**